packages/teams-js/src/public/dialog.ts

```diff
@@ -86,21 +98,30 @@ export namespace dialog {
   }
 
   /**
-   * Fucntion to call when an event is received from the Parent
+   * Register a listener that wil be triggerd when an event is received from the parent
```

Comment:
"triggered". Maybe change the comment to "Register a listener that will be triggered when a message is received from the app that opened the dialog".

I assume this should also only be called from the dialog. Should we make sure that this is also ensureInitialized(FrameContexts.task), or does it have to be this way for backwards compatibility?

Reply:
I am not very familiar with the frame contexts, but when we talked in the app SDK sync, Erin said it is fine to call ensureInitialized with no parameters (I hope I heard her right). 

Backward compatibility is also a good point, but we are changing it in the tasks file as well.
I have a PR up to add ensureInitialized in tasks file. If we know this should only work for FrameContexts.task, I can fix it in my PR as well.

Reply:
As far as I remember, this function is the "new" version of ParentAppWindow.addEventListener. I can't think of a reason that we would want anyone *except* a task/dialog context to register for messages from the parent app that created it. Could look at uses of ParentAppWindow or ask Yuri in office hours about it maybe?

Also okay to open a bug to investigate this if you don't want to hold up this PR.

Reply:
opened a new work item for this :)